api/python/.python-version

```diff
@@ -0,0 +1 @@
+3.11
```

Comment:
I'm not sure it makes sense to have it here

Reply:
i'm setting this to 3.9